lib/translator/inquiry.cc

```diff
@@ -567,8 +567,8 @@ StatusCode TranslateLogicalBlockProvisioningVpd(
 StatusCode InquiryToNvme(Span<const uint8_t> raw_scsi,
                          NvmeCmdWrapper& identify_ns_wrapper,
                          NvmeCmdWrapper& identify_ctrl_wrapper,
-                         uint32_t& alloc_len, uint32_t nsid,
-                         Span<Allocation> allocations) {
+                         uint32_t page_size, uint32_t nsid,
+                         Span<Allocation> allocations, uint32_t& alloc_len) {
```

Comment:
Changing param order so output param is at the end, which is also consistent with other command translation libraries